AuxTel/Standard-Operations/Daytime-Operations/latiss-daily-calibrations-BIAS-DARK-only-procedure.rst

```diff
@@ -0,0 +1,54 @@
+.. _BLOCK source code: https://github.com/lsst-ts/ts_config_ocs/blob/develop/Scheduler/observing_blocks_auxtel/block-295-latiss_daily_calibrations.json
+.. _AuxTel (LATISS) Temperatures and Pressures dashboard: https://summit-lsp.lsst.codes/chronograf/sources/1/dashboards/14
+
+.. |author| replace:: *Karla Peña Ramírez*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *OS team*
+
+.. _Daytime-Operations-LATISS-Daily-Calibrations-BIAS-DARK-only-Procedure:
+############################################
+LATISS Daily Calibrations BIAS and DARK only
+############################################
+
+.. _Daytime-Operations-LATISS-Daily-Calibrations-BIAS-DARK-only-Overview:
+Overview
+========
+This procedure will acquire BIAS and DARK calibrations images. After the entire script completion, the observer can decide whether to vent, proceed to on-sky or shutdown.
+This page assumes the reader is familiar with the content explained in the Observatory section: :ref:`Introduction to the combined calibrations generation procedure <Introduction-Combined-Calibrations-Generation-index>` 
+
+.. warning::
+  This procedure does not involves telescope motion with AuxTel. Announce that you are about to run the calibrations without telescope or dome movement on the *#summit-announce* and *#summit-auxtel* channels before you run the following steps.
+
+.. _Daytime-Operations-LATISS-Daily-Calibrations-BIAS-DARK-only-Post-Conditions:
+Post-Condition
+==============
+- Individual BIAS and DARK calibration images will be taken and verified using the `cp_verify`_ framework.
+- If ``generate_calibrations`` is ``True``, a (daily) combined calibration image per detector will be certified in a `butler`_ ``CALIBRATION`` `collection`_.
+
+.. _Daytime-Operations-LATISS-Daily-Calibrations-BIAS-DARK-only-Procedure-Steps:
+Procedure Steps
+===============
+#. Calibrations images should only be taken if the LATISS WREB temperature is under temperature control. Check the LATISS WREB temperatures under the `AuxTel (LATISS) Temperatures and Pressures dashboard`_ in Chronograf. WREB temperatures are visible in the lower middle panel labeled *WREB On Board*. If the *mean_temp2* (top blue line) is between 26-29 degress C, the temperature is suitable for taking calibrations. During the daytime, the fan on the WREB board may not be sufficient to cool the WREB down to these temperatures, so during warmer months you may have to wait until later in the day or early in the morning for it to reach the desired temperature. If the temperature is too high, do not proceed to the next steps. 
+#. Enable ATCS and LATISS using the standard scripts :file:`enable_atcs.py` and :file:`enable_latiss.py` with no configuration. 
+#. Enable the ``OCPS:2`` scheduler. Use the standard script :file:`auxtel/scheduler/enable.py` with the configuration required for the run and available in the corresponding night log. 
```

Comment:
This step should say to enable the OCPS:1 CSC, which is also needed to run the make_latiss_calibrations.py Script. You can do this using the set_summary_state script using a configuration like the one shown in the first line here https://confluence.lsstcorp.org/pages/viewpage.action?pageId=222743981.

Then, you should add an additional step to enable the ``Scheduler.2`` using the script as described here.

Reply:
Thank you Erik for the explanation. The original procedure was corrected and expanded.